agent/src/softwarecontaineragent.cpp

```diff
@@ -48,7 +48,6 @@ bool SoftwareContainerAgent::triggerPreload()
 inline bool SoftwareContainerAgent::isIdValid (ContainerID containerID)
 {
     return ((containerID < UINT32_MAX)
-            && (containerID >= 0)
```

Comment:
I might be wrong here, but -1 would be 0xFFFFFFFF in an unsigned case, which is >= 0.